x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/ilm/IndexLifecycleInitialisationTests.java

```diff
@@ -201,7 +201,14 @@ public void testExplainExecution() throws Exception {
         long actualModifiedDate = Instant.parse(responseItem.getModifiedDate()).toEpochMilli();
 
         logger.info("Creating index [test]");
-        CreateIndexResponse createIndexResponse = client().admin().indices().create(createIndexRequest("test").settings(settings))
+        CreateIndexResponse createIndexResponse = client().admin().indices()
+            .create(
+                createIndexRequest("test").settings(
+                    Settings.builder()
+                        .put(settings)
+                        .put(LifecycleSettings.SETTING_LIFECYCLE_ORIGINATION_DATE, 1000L)
```

Comment:
Can you add another check underneath this that setting this setting back to `null` goes back to using the original lifecycle creation date?